File: FitToFit/FitToFit.Model/Termini.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FitToFit.Model
{
    public class Termini
    {
        public int TerminId { get; set; }

        public string Dan { get; set; } = null!;

        public string Sat { get; set; } = null!;

        public int BrojClanova { get; set; }

        public int TreningId { get; set; }

        public int TrenerId { get; set; }

        public int SalaId { get; set; }
    }
}

File: FitToFit/UI/fittofit_mobile/lib/models/termini.dart
import 'package:json_annotation/json_annotation.dart';

part 'termini.g.dart';

@JsonSerializable()
class Termini {
  int terminId;
  String dan;
  String? sat;
  int brojClanova;
  int treningId;
  int trenerId;
  int salaId;

  Termini(
      {required this.terminId,
      required this.dan,
      this.sat,
      required this.brojClanova,
      required this.treningId,
      required this.trenerId,
      required this.salaId});

  factory Termini.fromJson(Map<String, dynamic> json) =>
      _$TerminiFromJson(json);

  Map<String, dynamic> toJson() => _$TerminiToJson(this);
}
